rust/agama-server/src/network/model.rs

```diff
@@ -663,6 +663,7 @@ pub enum ConnectionConfig {
     Vlan(VlanConfig),
     Bridge(BridgeConfig),
     Infiniband(InfinibandConfig),
+    Tun(TunConfig),
```

Comment:
I used `Tun` as name, cause it is the name in the NetworkManager dbus model as well (see: https://networkmanager.dev/docs/api/latest/nm-settings-dbus.html  search `tun setting`).

Reply:
Do not have an strong opinion, both looks good to me.